pom-explorer-core/src/main/java/fr/lteconsulting/pomexplorer/TransitivityResolver.java

```diff
@@ -306,4 +291,24 @@ private void buildDependencyTree( Queue<DependencyNode> nodeQueue, boolean full,
 			}
 		}
 	}
+
+	private static Optional<VersionScope> getVersionScopeFromDependencyManagement( DependencyNode node, DependencyKey dependencyKey, RawDependency dependency )
```

Comment:
This could be moved to a AnalyzerHelper class because it encapsulate a kind of "macro" use of the Analyzer and not adding a new functionality..

Reply:
that's not new logic, I only moved it to an own method